build/build.fsx

```diff
@@ -1,9 +1,17 @@
 #r "paket:
+frameworks: net6.0
+source https://api.nuget.org/v3/index.json
+nuget Microsoft.Build 17.3.2
+nuget Microsoft.Build.Framework 17.3.2
+nuget Microsoft.Build.Tasks.Core 17.3.2
+nuget Microsoft.Build.Utilities.Core 17.3.2
+nuget FSharp.Core ~> 6.0
 nuget Fake.IO.FileSystem
 nuget Fake.DotNet
 nuget Fake.DotNet.Cli
 nuget Fake.Tools.Git
 nuget Fake.Core.Target //"
+
```

Comment:
Empty line is needed. Fails otherwise. Sometimes I want to quit programming 🤷

Reply:
Spend a couple of hours on that 😠

Reply:
🤯 😭